### PR TITLE
fix(hmi-server): document service resources are unauthenticated

### LINKED ISSUE #1086: [BUG]: fix authentication on document resources
**Describe the bug**
The document service resources are not protected by Authentication - likely a copy/paste side effect of decommissioning the old document service

### PR DESCRIPTION
**Pull request by dvince2**
Adding the @Authenticated annotation to our document resources 

Resolves #1086

### CODE REVIEW

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/dataservice/ConceptResource.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
 import javax.ws.rs.core.Response;
 
 @Path("/api/concepts")
-@Authenticated
+
```
Comment (YohannParis) [inline packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/dataservice/ConceptResource.java:22]: Are you sure you want a white space here?

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/documentservice/DictionariesResource.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package software.uncharted.terarium.hmiserver.resources.documentservice;
 
 
+import io.quarkus.security.Authenticated;
```
Comment (YohannParis) [inline packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/documentservice/DictionariesResource.java:4]: Where this package being used?

**Review by YohannParis — COMMENTED** [review]
I don't really get what's happening here, you remove the annotation but you added the packages without the annotation.

**Review by YohannParis — APPROVED** [review]

**Comment by dvince2** [comment]
> 

Originally we had @Authenticated on like 80% of our resources.  This PR was initially to add it to the remaining 20%... however after the PR was issued I realized that by default quarkus has authentication enabled on everything and this annotation isn't needed at all. Rather than leave it to cause confusion, I removed it all together... but forgot to remove the import statements everywhere that it was :)

**Pull request merged**